### ISSUE TITLE
Plot the amount of data objects over the course of time

### ISSUE THREAD

**Issue by lukas-rotsching** [amosproj/amos2023ss02-open-search-meta-data-hub#18]
## User story
1. As a user
2. I want a graph that shows the amount of indexed data objects over the course of time
3. So that i have something nice to look at. I like charts that go up.

## Acceptance criteria
* Implement a plot that shows the number of data objects in the database over the course of time
* The plot is expected to automatically scale its dimensions to values that are reasonable for the available data (e.g. if the first object was added 10 days ago, i don't need a x-axis that shows the trend for the last 5 years. However, if i add new objects for 5 years daily, i want to see the development over 5 years and not only the last 3 months...)

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
